src/nci_util.c

```diff
@@ -308,6 +308,140 @@ nci_parse_iso_dep_poll_a_param(
     return FALSE;
 }
 
+static
+gboolean
+nci_parse_nfc_dep_poll_param(
+    NciActivationParamNfcDepPoll* param,
+    const guint8* bytes,
+    guint len)
+{
+    /* ATR_RES Length */
+    const guint8 atr_res_len = bytes[0];
+
+    /*
+     * NFCForum-TS-NCI-1.0
+     * Table 82: Activation Parameters for NFC-DEP Poll Mode
+     *
+     * +=========================================================+
+     * | Offset | Size | Description                             |
+     * +=========================================================+
+     * | 0      | 1    | Length of ATR_RES Command Parameter (n) |
+     * | 1      | n    | ATR_RES bytes from and including Byte 3 |
+     * +=========================================================+
+     */
+    if (atr_res_len >= 15 && len >= atr_res_len + 1) {
+        const guint8* atr_res = bytes + 1;
+
+        /*
+         * NFCForum-TS-DigitalProtocol-1.0
+         * 14.6.3 ATR_RES Response
+         */
+        memcpy(param->nfcid3, atr_res, sizeof(param->nfcid3));
```

Comment:
nit: sizeof is not a function and param->nfcid3 is not a type -> superfluous parentheses.